flake8_trio/runner.py

```diff
@@ -100,20 +100,20 @@ def visit(self, node: ast.AST):
 
 
 class Flake8TrioRunner_cst:
```

Comment:
the runner is kind of .. not necessary and clunky, esp as currently `__init__` does a bunch of logic itself. Might warrant some restructuring.